README.md

```diff
@@ -154,13 +155,25 @@ qrencode -t ANSI256 'otpauth://totp/VIP%20Access:VSSTXXXX?secret=YYYY&issuer=Sym
 
 Scan the code into your TOTP generating app, like Authy.
 
+
+### Using HOTP
+
+Use VSMB as token type `vipaccess provision -p -t VSMB`
+Known Limitations:
+* No way to re-synchronize if token ever gets out of sync with VIP Server.
+* You should use HEX code with Yubikeys
+* Tokens have an expiry date similar TOTP tokens unlike Yubikey VIP Tokens.
+* First try may not work as we used that token to validate it.
```

Comment:
What does this mean? Are you saying that when because we validate a HOTP token by reading the first code, the counter will be out of sync?

Reply:
We used that otp to validate the toke. Because its event based, OTP once used cannot be used again & first token generated by Yubikey or Google Authenticator does not work. Counter wont be out of sync as in unusable, but we just need to use from second otp. i.e counter=2

Reply:
About the Expiry date.
I am not sure how it works. Yubikeys were HOTP based and they do not have an expiry date. But the Expiry date is in the response similar to TOTP tokens. So the token might not work after that date.